File: app/assets/javascripts/bullseye.js
#= require jquery

this.Bullseye = {
  target: function(controller, action, callback) {
    this.targets[controller] = (this.targets[controller] || {});
    this.targets[controller][action] = callback;
  },

  targets: {},

  exec: function(controller, action) {
    if (this.targets[controller] && this.targets[controller][action]) {
      this.targets[controller][action].apply(this.context);
    }
  }
};

Bullseye.context = this;

$(function() {
  Bullseye.exec($('body').data('controller'), $('body').data('action'));
});

File: skel/config/initializers/bullseye.rb
Bullseye.configure do |c|
  c.js_controller_search = %{$('body').data('controller')}
  c.js_action_search = %{$('body').data('action')}

  c.css_selector = %{body[data-action=':action'][data-controller=':controller']}
  c.html_tag = { 'data-action' => ':action', 'data-controller' => ':controller' }

  c.fuzzy_search = false
end
